Created recipe for stasis chamber, added slab and stair variants to several lab building blocks.

bijimio opened the pull request:
## Other Proposed Changes:

- Swapped darkblue tiles for teal tiles, as the darkblue tiles are no longer seen in the latest release of the Changed game.

- Added green counterparts for the striped and tiled walls.

bijimio committed:
Bijimio Experimental Fork V7.5.8 | Created recipe for stasis chamber, added slab and stair variants to several building blocks.

bijimio committed:
Merge branch 'LtxProgrammer:14.1-features' into 14.1-features

bijimio committed:
Bijimio Experimental Fork V7.5.9 | Fixed slab drop logic.

bijimio committed:
Bijimio Experimental Fork V7.6.0 | Added more stair and slab variants, Updated darkblue tiles to teal tiles.

bijimio committed:
Bijimio Experimental Fork V7.6.1 | Added green variants of striped and tiled walls

Foxyas commented:
@bijimio can you add this change? i found a misspixel in gas wolf texture

![gas_wolf](https://github.com/user-attachments/assets/2f1d6406-7a5c-460e-b95d-492d046b3c92)

Foxyas commented:
is in the neck part

bijimio commented:
> is in the neck part
was the error on the head or torso part?

Foxyas commented:
> > is in the neck part
> was the error on the head or torso part?

Up top of torso the gas_wolf.png have a miss pixel in there

bijimio commented:
just checked everything and there doesn't appear to be any gaps when i combine the textures.

was it the light or dark variation?

Foxyas commented:
> just checked everything and there doesn't appear to be any gaps when i combine the textures.
> 
> was it the light or dark variation?
> 

Isn't in the coat but inside of the fur thing

Foxyas commented:
![Screenshot_20250514_013805_Chrome.jpg](https://github.com/user-attachments/assets/5c4fcda8-1689-40d6-82a8-431654bbb93f)

![Screenshot_20250514_013812_Chrome.jpg](https://github.com/user-attachments/assets/9a57a37c-1311-4794-a340-ef5b8d9192b1)

Foxyas commented:
I check the 0.14.1 breach and you already fixed that :p
![Screenshot_20250514_014105_Chrome.jpg](https://github.com/user-attachments/assets/b51ec7f1-1634-40fb-9471-f1fb0dd220e7)

LtxProgrammer committed:
Removed unused line in DF

Though it's not documented well, `BLOCK_ID_REMAP` is for blocks that don't have an item that pairs with it. The entry in `BLOCK_ITEM_ID_REMAP` is sufficient to fix the data.
